src/System.Threading.Timer/System.Threading.Timer.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2013
```

Comment:
Copied the csproj and sln from a different project...strange the VS 2015 didn't update this value when I launched this sln